src/auth-service/utils/email.msgs.js

```diff
@@ -19,6 +19,9 @@ module.exports = {
       `Dear ${firstName + " " + lastName} \n\n` +
       "Your request to join the AirQo platform has been received.\n" +
       "We shall get back to you as soon as possible. \n\n" +
+      "Before utilising airqo data, your application record has to undergo \n" +
+      "the process of approval by platform administration. Once your \n" +
+      "application reviewed, you will receive an email on the mailbox \n\n" +
```

Comment:
we might be missing a word here! ...once your application *is* reviewed...

Reply:
Resolved @danielogen